public-api/src/diff.rs

```diff
@@ -142,10 +142,14 @@ fn bag_to_path_map<'a>(difference: impl Iterator<Item = (&'a PublicItem, usize)>
 
 #[cfg(test)]
 mod tests {
+    use rustdoc_types::Id;
+
     use crate::tokens::Token;
 
     use super::*;
 
+    static DUMMY_ID: Id = Id(1234);
```

Comment:
I think it works to have
```suggestion
    const DUMMY_ID: Id = Id(1234);
```
?

Reply:
:man_facepalming: